lang/python/python-astral/Makefile

```diff
@@ -30,7 +30,8 @@ define Package/python3-astral
   URL:=https://github.com/sffjunkie/astral
   DEPENDS+= \
       +python3-light \
-      +python3-pytz
+      +python3-pytz \
+      +python3-distutils
```

Comment:
It appears that monkeypatch.py is only used in [setup.py](https://github.com/sffjunkie/astral/blob/2.2/setup.py#L7), so I don't think distutils is required at run-time.

Reply:
Fixed in https://github.com/openwrt/packages/pull/16541/commits/071e66b9601e7379f5f67144fd05ec4633e0aad7